v3/lib/endpoints/class-acf-to-rest-api-controller.php

```diff
@@ -109,6 +114,26 @@ public function update_item_permissions_check( $request ) {
 			return apply_filters( 'acf/rest_api/item_permissions/update', current_user_can( 'edit_posts' ), $request, $this->type );
 		}
 
+		public function delete_item_permissions_check( $request ) {
+			return apply_filters( 'acf/rest_api/item_permissions/delete', current_user_can( 'delete_posts' ), $request, $this->type );
+		}
+
+		public function delete_item( $request ) {
+			$fields = $this->acf->get_fields( $request )['acf'];
```

Comment:
Please, check before to access.
```
$fields = $this->acf->get_fields( $request );
if ( $fields && isset( $fielsd['acf'] ) ) {
	$fields = $fields['acf'];
}
```

Reply:
Fixed in e2f5c9c1b65f0402fcd989bd7fd90a8e87da253b